src/viewers/VivViewer.jsx

```diff
@@ -302,7 +302,7 @@ class VivViewerWrapper extends PureComponent {
         // eslint-disable-next-line react/jsx-props-no-spreading
         {...(deckProps ?? {})}
         layerFilter={this.layerFilter}
-        layers={this._renderLayers()}
+        layers={[...this._renderLayers(), ...deckProps.layers]}
```

Comment:
`deckProps.layers` (or even `deckProps`) could be null.  Could you add checks for this?

Reply:
I think something similar to the above `deckProps ?? {}` (perhaps refactored to avoid duplicating that code) plus a check on `.layers` would suffice.  Additionally, please add documentation to all of the components that wrap `VivViewer` that this is enabled now.

Reply:
I added a check for undefined, in case, `deckProps` or `deckProps.layers `does not get passed in at all.

Reply:
Please let me know if any additional changes are needed.